bootstrap/src/main/java/io/airlift/bootstrap/Bootstrap.java

```diff
@@ -78,6 +78,7 @@ private enum State { UNINITIALIZED, CONFIGURED, INITIALIZED }
     private Map<String, String> requiredConfigurationProperties;
     private Map<String, String> optionalConfigurationProperties;
     private boolean initializeLogging = true;
+    private boolean envInterpolation;
```

Comment:
This should be enabled by default to maintain backward compatibility. Environment interpolation is a feature of Airlift that's useful when building services (the primary use case). Any other cases should explicitly opt out.

Reply:
Done